contractsV5/FeeHandler.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity 0.5.11;
+
+import "./IKyberDAO.sol";
+import "./KyberNetwork.sol";
+import "./IFeeHandler.sol";
+import "./PermissionGroupsV5.sol";
+import "./UtilsV5.sol";
+
+contract FeeHandler is IFeeHandler, Utils {
+
+    IKyberDAO public kyberDAOContract;
+    KyberNetwork public kyberNetworkContract;
```

Comment:
here we want the interface.
or address.

otherwise you include the whole network code and deploy it as part of your code.

Reply:
I need to call the variable `reserveRebateWallets` that does not exist in the interface.

Reply:
Okay I understand what you mean now. We can declare a function in the interface for the the public variable like the `decimals()` function in the ERC20 interface. 

Do we want to add this into the `IKyberNetwork.sol` or a separate interface?

Reply:
why does fee handle need to call reserveRebateWallets? its getting the wallet list.
its doesn't know about the reserves at all. only the wallets.

Reply:
It needs to know the `reserveRebateWallet` because in the `claimReserveRebate` function, the parameter passed into the function is a reserve address, not a rebate wallet address. I'm planning to change it to a rebate wallet address instead (assuming that the DAO has access to the KyberNetwork contract)

Reply:
I've changed it to rebateWallet instead of using reserve address.

Reply:
so please remove the include for kyberNetwork
and also see if u can define kyber network as address parameter.